pkg/cmd/show.go

```diff
@@ -113,7 +116,10 @@ func newShowCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 			if err != nil {
 				return err
 			}
-			fmt.Fprint(out, output)
+			_, err = fmt.Fprint(out, output)
+			if err != nil {
+				return err
+			}
```

Comment:
`out` is the output (e.g. stdout). Today the error is ignored. This means that the Helm CLI can be run detached from a terminal. This change would cause an error if Helm is run detached from a terminal.

This may cause a problem. Do we want to exclude that use case? I've not used it before but I can imagine someone might.

In any case, this is a change in behavior. Credit to the gophers slack. Discussing this topic there brought up this angle.